extension/src/test/LanguageFunctions.test.ts

```diff
@@ -76,7 +76,9 @@ suite("Language Functions Tests", function () {
     const settings = SettingsLoader.getSettings();
     const foundMatch = await LanguageFunctions.findNextUntranslatedText(
       WorkspaceFunctions.getLangXlfFiles(settings, appManifest),
-      false
+      false,
```

Comment:
No actual test of the new feature or am I missing something?

Reply:
sorry!